test/functional/content_view/version/update_test.rb

```diff
@@ -0,0 +1,97 @@
+require_relative '../../test_helper'
+require 'hammer_cli_katello/content_view_version'
+
+module HammerCLIKatello
+  describe ContentViewVersion::UpdateCommand do
+    include OrganizationHelpers
+    it 'allows minimal options' do
+      ex = api_expects(:content_view_versions, :update).with_params('description' => 'pizza')
+      ex.returns('id' => '2', 'description' => 'pizza')
+
+      result = run_cmd(%w(content-view version update --id 2 --description pizza))
+      assert_equal(result.exit_code, 0)
+    end
+  end
+
+  describe 'content view options' do
+    it 'allows content view id' do
+      ex = api_expects(:content_view_versions, :update).with_params('description' => 'pizza')
+      ex.returns('id' => '2', 'description' => 'pizza', 'content_view' => [{'id' => '2'}])
+
+      # rubocop:disable LineLength
+      result = run_cmd(%w(content-view version update --id 2 --description pizza --content-view-id 2))
```

Comment:
This isn't really a normal usage of the command. `--content-view-id` is supposed to be for resolving content view version ID from the version string. By using `--id`, `--content-view-id` isn't needed.